build/Wikibase/Dockerfile

```diff
@@ -17,10 +17,6 @@ RUN git clone https://gerrit.wikimedia.org/r/mediawiki/extensions/Wikibase /tmp/
 FROM ${MEDIAWIKI_IMAGE_URL} as collector
 COPY --from=git-checkout /tmp/wikibase /var/www/html/extensions/Wikibase
 
-COPY ./fix-do-not-add-stdport-in-detectServer.patch /tmp/fix-do-not-add-stdport-in-detectServer.patch
```

Comment:
What was this for previously?

Reply:
Applying Robert's patch, which fixed the port number bug. WebRequest.php has been updated such that the patch now causes build to fail, and the issue it was patching has been resolved

Reply:
Got it. So this was a new patch with 1.41.0, now fixed in 1.41.1